src/index.d.ts

```diff
@@ -80,7 +80,7 @@ declare function VerifierSync(token: string | Buffer): any
 declare function VerifierAsync(token: string | Buffer): Promise<any>
 declare function VerifierAsync(token: string | Buffer, cb: object): void
 
-export interface JwtHeader {
+export interface JwtHeader extends Record<string, any> {
```

Comment:
Did you check if this still gives code hints?

Reply:
Yes it still does!